Python/bytecodes.c

```diff
@@ -2052,7 +2055,8 @@ dummy_func(
 
         macro(LOAD_ATTR_CLASS) =
             unused/1 +
-            _CHECK_ATTR_CLASS +
```

Comment:
Is it worth breaking this up?
It will slow down code that needs it, and it looks like the optimizer will remove both of the parts in most cases.